src/view.js

```diff
@@ -0,0 +1,42 @@
+import Vue from 'vue'
```

Comment:
Can we find a better name for this and related files? It's not really clear why is this used and what it does

Reply:
Maybe `component-view`?

Reply:
Mhm, that's better

Reply:
Yep, that's better